data/vyrmeid/vyrmeid.txt

```diff
@@ -201,6 +201,7 @@ ship "Astral Cetacean"
 		"ramscoop" 5
 		"energy capacity" 2000
 		"energy generation" 160
+		"inscrutable" 1
```

Comment:
Put this at the bottom of the first block of attributes (right after engine capacity).

Reply:
Done.